terraform/testcases/statsd_otellib_instScopeEnabled/parameters.tfvars

```diff
@@ -0,0 +1,17 @@
+# this file is defined in validator/src/main/resources/validations
+validation_config = "statsd-otellib-metric-validation.yml"
+
+# sample application image to emit the trace data
+sample_app = "statsd"
+
+# data type will be emitted. Possible values: metric or trace
+soaking_data_mode = "metric"
+
+# data model type. possible values: otlp, xray, etc
+soaking_data_type = "statsd"
+
+feature_gate = "+aws.statsd.populateInstrumentationScope"
+
+aoc_image_repo = "public.ecr.aws/aws-otel-test/adot-collector-integration-test"
+
+aoc_version = "v0.30.0-9468cd0"
```

Comment:
This needs to be removed. I had committed the change already but it looks like you force pushed over it. Next time pull changes down from your branch before force pushing.

Reply:
In hindsight, I should have let you commit the changes. I will not do that in the future, in my head this would have made it easier for you but I see how it could be confusing.

Reply:
> This needs to be removed. I had committed the change already but it looks like you force pushed over it. Next time pull changes down from your branch before force pushing.

I agree with that yes. I shall take latest on the branch.